<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="control_panel">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Control Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis1_label">
        <property name="text">
         <string>Axis 1 Control</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axis1">
        <item>
         <widget class="QSlider" name="axis1_slider">
          <property name="maximum">
           <number>36</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>18</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="axis1_box">
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="axis2_label">
        <property name="text">
         <string>Axis 2 Control</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axis2">
        <item>
         <widget class="QSlider" name="axis2_slider">
          <property name="maximum">
           <number>36</number>
          </property>
          <property name="value">
           <number>18</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="axis2_box">
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="axis3_label">
        <property name="text">
         <string>Axis 3 Control</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axis3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="axis3_slider">
          <property name="maximum">
           <number>36</number>
          </property>
          <property name="value">
           <number>18</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="axis3_box">
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="axis4_label">
        <property name="text">
         <string>Axis 4 Control</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axis4">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="axis4_slider">
          <property name="maximum">
           <number>36</number>
          </property>
          <property name="value">
           <number>18</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="axis4_box">
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="axis5_label">
        <property name="text">
         <string>Axis 5 Control</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axis5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="axis5_slider">
          <property name="maximum">
           <number>36</number>
          </property>
          <property name="value">
           <number>18</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="axis5_box">
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="claw_label">
        <property name="text">
         <string>Claw Control</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="claw_control">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="claw_pos">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="claw_pos_label">
            <property name="text">
             <string>Claw Position Closed (in %)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="claw_pos_box">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="claw_control_buttons">
          <item>
           <widget class="QPushButton" name="claw_open">
            <property name="text">
             <string>Open Claw</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="claw_close">
            <property name="text">
             <string>Close Claw</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="resetRobot">
        <property name="text">
         <string>Reset Robot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speed_label">
        <property name="text">
         <string>Movement Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="speed">
        <item>
         <widget class="QSlider" name="speed_slider">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="speed_box">
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="waypoint_panel">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="waypoint_menu">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Waypoint Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="waypoint_control">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="add_waypoint">
          <property name="text">
           <string>Add Waypoint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_last_waypoint">
          <property name="text">
           <string>Remove Last Waypoint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copy_Waypoints">
          <property name="text">
           <string>Copy Waypoints</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="serial_port_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Serial Port:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="serial_port_box">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>/dev/ttyACM0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reconnect">
          <property name="text">
           <string>Reconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="serial_status">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
